annotation_import/prompt_response.ipynb

```diff
@@ -0,0 +1,608 @@
+        ")\n",
+        "\n",
+        "nested_response_checklist_annotation = lb_types.ClassificationAnnotation(\n",
+        "    name=\"nested_response_checklist_question\",\n",
+        "    value=lb_types.Checklist(answer=[\n",
+        "        lb_types.ClassificationAnswer(\n",
+        "            name=\"first_checklist_answer\",\n",
+        "            classifications=[\n",
+        "                lb_types.ClassificationAnnotation(\n",
+        "                    name=\"sub_checklist_question\",\n",
+        "                    value=lb_types.Checklist(answer=[\n",
+        "                        lb_types.ClassificationAnswer(\n",
+        "                            name=\"first_sub_checklist_answer\")\n",
+        "                    ]),\n",
+        "                )\n",
+        "            ],\n",
+        "        )\n",
+        "    ]),\n",
+        ")\n",
+        "\n",
+        "nested_response_radio_annotation_ndjson = {\n",
+        "    \"name\":\n",
+        "        \"nested_radio_question\",\n",
+        "    \"answer\": [{\n",
+        "        \"name\":\n",
+        "            \"first_radio_answer\",\n",
+        "        \"classifications\": [{\n",
+        "            \"name\": \"sub_radio_question\",\n",
+        "            \"answer\": {\n",
+        "                \"name\": \"first_sub_radio_answer\"\n",
+        "            },\n",
+        "        }],\n",
+        "    }],\n",
+        "}\n",
+        "\n",
+        "nested_response_checklist_annotation_ndjson = {\n",
+        "    \"name\":\n",
+        "        \"nested_checklist_question\",\n",
+        "    \"answer\": [{\n",
+        "        \"name\":\n",
+        "            \"first_checklist_answer\",\n",
+        "        \"classifications\": [{\n",
+        "            \"name\": \"sub_checklist_question\",\n",
+        "            \"answer\": {\n",
+        "                \"name\": \"first_sub_checklist_answer\"\n",
+        "            },\n",
+        "        }],\n",
+        "    }],\n",
+        "}"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## Step 1: Create a project and data rows using the Labelbox UI\n",
+        "\n",
+        "Each type of the prompt and response generation project requires different setup. See [prompt and response project](https://docs.labelbox.com/reference/prompt-and-response-projects) for more details on the differences.\n",
+        "\n",
+        "In this tutorial, we will show how to import annotations for a prompt and response creation (humans generate prompts and responses) project. The process is also similar for prompt creation (humans generate prompts) and response creation (humans generate responses to uploaded prompts) projects. See [import prompt and response annotations](https://docs.labelbox.com/reference/import-prompt-and-response-annotations) for a tutorial and more examples on other project types."
```

Comment:
Though it looks less concise, I added the exact names on the UI since this section is about creating the project using the UI.
The URL "https://docs.labelbox.com/reference/import-prompt-and-response-annotations" is broken. Is "https://docs.labelbox.com/reference/import-prompt-and-response-data" the correct one?

Reply:
Im swapping this out to import-prompt-and-response-annotations from a doc change we did PR
https://github.com/Labelbox/labelbox-documentation/pull/87
Once we switch slug it should work. I guess we can switch slug now